Dockerfile

```diff
@@ -53,9 +44,9 @@ LABEL alpine-version=$alpine_version
 COPY --from=install /install /
 
 # Default to UTF-8 file.encoding
-ENV LANG en_US.UTF-8
-ENV LANGUAGE en_US:en
-ENV LC_ALL en_US.UTF-8
+ENV LANG=en_US.UTF-8
```

Comment:
Thanks, didn't now that

https://docs.docker.com/reference/build-checks/legacy-key-value-format/